staging/src/k8s.io/client-go/util/workqueue/queue.go

```diff
@@ -208,6 +218,8 @@ type Typed[t comparable] struct {
 
 	metrics queueMetrics[t]
 
+	cancel                     context.CancelFunc
+	wg                         *sync.WaitGroup
```

Comment:
same here: would expect go doc what these are for, how they interact.

Reply:
Added, let me know if this is what you had in mind.